support/ccm.rb

```diff
@@ -458,12 +459,7 @@ def start(jvm_arg=nil)
         attempts = 1
 
         begin
-          if jvm_arg
-            @ccm.exec('start', '--wait-other-notice', '--wait-for-binary-proto', "--jvm_arg=#{jvm_arg}")
-          else
-            @ccm.exec('start', '--wait-other-notice', '--wait-for-binary-proto')
-          end
-
+          @ccm.exec('start', '--wait-for-binary-proto', jvm_arg ? "--jvm_arg=#{jvm_arg}" : '')
```

Comment:
What happened to the wait-other-notice arg? Was that intentionally removed here?